src/test/scala/org/camunda/feel/impl/builtin/BuiltinFunctionsTest.scala

```diff
@@ -243,4 +243,27 @@ class BuiltinFunctionsTest
       "Assertion failure on evaluate the expression 'list contains(assert(my_list, my_list != null, \"The condition is not true\"), 2)': The condition is not true"
     )
   }
+
+  "A is blank() function" should "return Boolean" in {
```

Comment:
:x: Same as the implementation, we should add the test cases to [BuiltinStringFunctionsTest](https://github.com/camunda/feel-scala/blob/main/src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala).